lib/iex/errors/symbol_not_found_error.rb

```diff
@@ -1,14 +1,27 @@
 module IEX
   module Errors
-    class SymbolNotFoundError < StandardError
-      attr_reader :symbol
+    class IexError < StandardError
```

Comment:
Should this be IEXError? Or just Error since it's already in the namespace?

Reply:
I like just `Error`, will update.